### PR TITLE
complete checks > complete check

### PR DESCRIPTION
**Pull request by BrodyHughes**
Fixes APP-1189

complete checks > complete check

### CODE REVIEW

**Review by walmat — APPROVED** [review]
Legendary stuff man 👑

**Comment by BrodyHughes** [comment]
> Legendary stuff man 👑

This was a tough one

File: src/components/send/SendButton.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default function SendButton({
       isAuthorizing={isAuthorizing}
       label={
         disabled && requiresChecks
-          ? `􀄨 ${lang.t('wallet.transaction.complete_checks')}`
+          ? `􀄨 ${lang.t('wallet.transaction.complete_check')}`
```
Comment (skylarbarrera) [inline src/components/send/SendButton.js:42]: this could be plural with ENS p sure, we may need to count the checks

Comment (BrodyHughes) [reply src/components/send/SendButton.js:42]: ahh yeah i'll look at it

Comment (BrodyHughes) [reply src/components/send/SendButton.js:42]: @skylarbarrera 
All the ENS checks are optional so it actually doesn't apply here. 
Screenshot 2024-04-17 at 10 29 30 AM

Comment (BrodyHughes) [reply src/components/send/SendButton.js:42]: also i forgot about this PR lol

**Pull request merged**